packages/KIcon/icons/index.js

```diff
@@ -5,6 +5,8 @@ import arrowUp from './icn-arrow-up.svg'
 import back from './icn-back.svg'
 import book from './icn-book.svg'
 import brain from './icn-brain.svg'
+import calendar from './icn-calendar.svg'
```

Comment:
Why are we adding 2 different calendar icons? 

For the sake of not having a mismatch all over Konnect, can we stick with one?

Reply:
We will eventually have just one, i wanted to demo what the two look like to PM/UX